docs/api/vulnerability-info.md

```diff
@@ -33,6 +33,16 @@ The data file schema does not allow for modification or redaction of known vulne
 Therefore if a server's vulnerability data source (for example the [GitHub Advisories Database](https://github.com/advisories)) modifies an existing advisory, the NuGet server must modify the page that the vulnerability information was previously reported.
 One way to achieve this with the suggested partition scheme is to treat all vulnerability modifications and deletions as a trigger to regenerate the complete `base.json` file, and empty `updates.json`.
 
+## Reusing nuget.org's vulnerability data
+
+NuGet does not enforce that resources in the [service index](./service-index.md), or [the vulnerability index](#vulnerability-index), must be on the same server as the service index itself.
+However, there are several reasons why some companies choose to block api.nuget.org at the firewall.
+To avoid connectivity issues, we recommend serving vulnerability data from your own web app, so that NuGet clients only make HTTP connections to a single host.
```

Comment:
```suggestion
To avoid connectivity issues, we recommend serving vulnerability data from your own web app, so that NuGet clients only make HTTP connections to a host you control.
```

Reply:
In the case of a .NET developer using a self-hosted feed, it's the feed implementation that needs to implement vulnerability info, but they don't control the server that it's installed on.   I'll push a change soon with different wording, so I'd like your opinion once it's pushed.